nx.json

```diff
@@ -0,0 +1,37 @@
+{
+    "tasksRunnerOptions": {
+        "default": {
+            "runner": "nx/tasks-runners/default",
+            "options": {
+                "cacheableOperations": ["build", "lint", "test", "e2e"]
+            }
+        }
+    },
+    "extends": "nx/presets/npm.json",
+    "$schema": "./node_modules/nx/schemas/nx-schema.json",
+    "generators": {
```

Comment:
I am not sold on having these generators here. There is very little boilerplate code needed in any of the packages and I fear this may incentivise code bloat